File: src/main/java/com/example/ssauc/user/contact/dto/Chatbot.java
package com.example.ssauc.user.contact.dto;

import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Chatbot {
    private String sender;   // "USER" or "BOT"
    private String message;  // 메시지 내용
}

File: src/main/java/com/example/ssauc/user/contact/dto/Chatbot.kt
package com.example.ssauc.user.contact.dto

import lombok.*

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
class Chatbot {
    public val sender: String? = null // "USER" or "BOT"
    public val message: String? = null // 메시지 내용
}